### PR TITLE
Add CUDA 10.0 version whl support

### PR DESCRIPTION
**Pull request by JanuszL**
Signed-off-by: Janusz Lisiecki <jlisiecki@nvidia.com>

### CODE REVIEW

**Comment by JanuszL** [comment]
CUDA 9 build 651652 
CUDA 10 build 651651

**Comment by cliffwoolley** [comment]
LGTM.  Let's check the URLs with Pramod as I'd mentioned.

File: README.rst [hunk 1]
```diff
@@ -55,28 +55,42 @@ Prerequisites
...
+
 .. code-block:: bash
 
-   pip install --extra-index-url https://developer.download.nvidia.com/compute/redist nvidia-dali
+   pip install --extra-index-url https://developer.download.nvidia.com/compute/redist/cuda/10.0 nvidia-dali
 
 .. note::
-   nvidia-dali package contains prebuilt versions of the dali tensorflow plugin for several versions of tensorflow.
-   Since release 0.6.1 there is also a possibility to install dali tensorflow plugin for the currently installed version of tensorflow, thus allowing forward compatibility:
+   nvidia-dali package contains prebuilt versions of the dali TensorFlow plugin for several versions of TensorFlow.
```
Comment (Kh4L) [inline README.rst]:
```suggestion
   nvidia-dali package contains prebuilt versions of the DALI TensorFlow plugin for several versions of TensorFlow.
```

**Comment by JanuszL** [comment]
Agreed with @cliffwoolley

**Pull request merged**